README

```diff
@@ -45,33 +41,43 @@ list of the people who have recently contributed, in alphabetical order:
    * Gabriel Altay          gabriel.altay@gmail.com
    * James Bordner          jobordner@ucsd.edu
    * Greg Bryan             gbryan@astro.columbia.edu
+   * Corey Brummel-Smith    cdbs3@gatech.edu
    * Iryna Butsky           ibutsky@uw.edu
    * Renyue Cen             cen@astro.princeton.edu
+   * Duncan Christie        duncanchristie@gmail.com
    * Dave Collins           dcollins4096@gmail.com
+   * Lauren Corlies         lcorlies@lsst.org
    * Brian Crosby           crosby.bd@gmail.com
    * Philipp Edelmann	    pedelmann@mpa-garching.mpg.de
-   * Andrew Emerick	    aemerick11@gmail.com
+   * Andrew Emerick         aemerick11@gmail.com
+   * Forrest Glines         forrestglines@gmail.com
    * Nathan Goldbaum        ngoldbau@ucsc.edu
    * Philipp Grete          grete@pa.msu.edu 
    * John Forbes            jcforbes@ucsc.edu
+   * Yusuke Fujimoto        yusuke.fujimoto.jp@gmail.com
    * Oliver Hahn            hahn@phys.ethz.ch
    * Robert Harkness        harkness@sdsc.edu
    * Elizabeth Harper-Clark h-clark@astro.utoronto.ca
    * Cameron Hummels        chummels@gmail.com
+   * Peter Iannucci         iannucii@mit.edu
    * Ji-hoon Kim            mornkr@tapir.caltech.edu
    * Daegene Koh            dkoh30@gatech.edu
-   * Shuo Kong		    skong.astro@gmail.com
+   * Shuo Kong              skong.astro@gmail.com
+   * Claire Kopenhafer      clairekope@gmail.com
    * Alexei Kritsuk         akritsuk@ucsd.edu
    * Michael Kuhlen         kuhlen@gmail.com
    * James Larrue           james.larrue@diopolis.com
    * Eve Lee                elee@cita.utoronto.ca
-   * Miao Li		    ml3322@columbia
+   * Miao Li                ml3322@columbia.edu
    * Yuan Li                yuan@astro.columbia.edu
```

Comment:
Assuming the FDM module is accepted, can we add it's author:

```suggestion
    * Yuan Li                yuan@astro.columbia.edu
    * Xinyu Li               xinyu.li@columbia.edu
```

Reply:
Done in a separate commit